ipalib/plugable.py

```diff
@@ -518,7 +518,7 @@ def build_global_parser(self, parser=None, context=None):
             help='Set environment variable KEY to VAL',
         )
         parser.add_option('-c', dest='conf', metavar='FILE',
-            help='Load configuration from FILE',
+            help='Load configuration from FILE. This must be an absolute path',
```

Comment:
PEP8 check fails for this line, but it fits the code style of the surrounding code. IMO this should be fixed as a separate issue.